testing/simapp/app.go

```diff
@@ -95,7 +95,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/server/config"
 	servertypes "github.com/cosmos/cosmos-sdk/server/types"
 	"github.com/cosmos/cosmos-sdk/std"
-	"github.com/cosmos/cosmos-sdk/testutil/testdata/testpb"
+	testdata_pulsar "github.com/cosmos/cosmos-sdk/testutil/testdata/testpb"
```

Comment:
what a weird ass linting error though

Reply:
Yeah, underscores are verboten aren't they? I fixed this in another branch, must have slipped through from merge conflicts